File: regiondeployer/configadapter.fs
module regiondeployer

open MoonMachine.credentials
open Microsoft.FSharp.Linq
open System
open MoonMachine.Infrastructure.models
open MoonMachine.Core.interop.models

let public initialise() : unit =
    #if DEBUG
        ProjectCredentials.get.globaloverride(new Nullable<mmenvironment>(mmenvironment.debug));
    #else
        ProjectCredentials.get.globaloverride(new Nullable<mmenvironment>(mmenvironment.prod));
    #endif

File: regiondeployer/ConfigAdapter.cs
using MoonMachine.credentials;
using Microsoft.FSharp.Linq;
using System;
using MoonMachine.Infrastructure.models;
using MoonMachine.Core.interop.models;

public class ConfigAdapter
{
    public void Initialise()
    {
        #if DEBUG
            ProjectCredentials.Get.GlobalOverride(new Nullable<mmenvironment>(mmenvironment.debug));
        #else
            ProjectCredentials.Get.GlobalOverride(new Nullable<mmenvironment>(mmenvironment.prod));
        #endif
    }
}